src/main/scala/codacy/brakeman/Brakeman.scala

```diff
@@ -202,8 +207,8 @@ object Brakeman extends Tool {
         Seq("-t", patternsIds.mkString(","))
     }
 
-    Seq("brakeman", "-f", "json") ++ patternsToTest ++ Seq(path.toString)
+    Seq("brakeman", "-A", "-f", "json") ++ patternsToTest ++ Seq(path.toString)
```

Comment:
I cannot find the `-A` option in here: https://github.com/presidentbeef/brakeman/wiki/Brakeman-Commandline-Options
Did you meant `-a`?